Client/Ecosia/UI/PageAction/PageActionMenu.swift

```diff
@@ -208,11 +211,45 @@ extension PageActionMenu: NotificationThemeable {
 // MARK: Cell
 
 class PageActionMenuCell: UITableViewCell {
+    private weak var badge: UIView?
+    
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    func isNew(_ isNew: Bool) {
+        if isNew {
+            if badge == nil {
+                let badge = UIView()
+                badge.translatesAutoresizingMaskIntoConstraints = false
+                badge.isUserInteractionEnabled = false
+                badge.backgroundColor = .theme.ecosia.primaryBrand
```

Comment:
imho it's necessary to define the theme sensitive colors independently of `badge==nil` - otherwise they won't get updated by theme changes at runtime,

Reply:
Not necessary, as the table view itself gets reloaded on theme changes.

Reply:
Good call, updated the badge for when the table get's reloaded but with different theme.